backend/src/schema/types/type/User.gql

```diff
@@ -214,10 +214,21 @@ type Mutation {
 
   muteUser(id: ID!): User
   unmuteUser(id: ID!): User
+
   blockUser(id: ID!): User
   unblockUser(id: ID!): User
 
+  followUser(id: ID!): User
+  unfollowUser(id: ID!): User
+
   switchUserRole(role: UserRole!, id: ID!): User
 
   saveCategorySettings(activeCategories: [String]): Boolean
+
+  requestPasswordReset(email: String!): Boolean!
+  resetPassword(email: String!, nonce: String!, newPassword: String!): Boolean!
+  changePassword(oldPassword: String!, newPassword: String!): String!
+
+  # Get a JWT Token for the given Email and password
+  login(email: String!, password: String!): String!
```

Comment:
Got an error on my local machine:
```
{
  extensions: {code: "GRAPHQL_VALIDATION_FAILED"},
  message: "Field \"login\" of type \"User\" must have a selection of subfields. Did you mean \"login { ... }\"?",
}
```

Reply:
I cannot reproduce this error

Reply:
![image](https://github.com/Ocelot-Social-Community/Ocelot-Social/assets/1238238/fa233217-5cba-48b8-913a-8e9dde296d67)

having it running locally (only neo4j in docker) in compiled version (`yarn build` and `yarn start` both in webapp and backend). Login works as expected.

**I cannot reproduce this error either**